--- sites/Trello/trello.app.js	2015-04-06 16:32:02.000000000 -0700
+++ fixed_for_Safari.js	2015-04-06 19:10:55.000000000 -0700
@@ -22078,17 +22078,17 @@
                     return e.getAsString(function(e) {
                         return t(null, e)
                     })
                 }, function(e, n) {
                     return t(n)
-                })) : t((null != (s = u.call(null != n ? n.types : void 0, "text/x-moz-url") >= 0) ? s : []) ? [n.getData("text/plain")] : [])
+                })) : t((null != (s = u.call(null != n ? n.types : void 0, "text/plain") >= 0) ? s : []) ? [n.getData("text/plain")] : [])
             }, e.prototype._isDraggingFiles = function(e) {
                 var t, n, r, i;
                 return t = null != (n = e.originalEvent) ? n.dataTransfer : void 0, null != t && (null != t.items && c.any(t.items, function(e) {
                     return "file" === e.kind || "text/uri-list" === e.type
                 }) || null != t.types && c.any(t.types, function(e) {
-                    return "application/x-moz-file" === e || "text/x-moz-url" === e
+                    return "application/x-moz-file" === e || "text/x-moz-url" === e || "text/uri-list" === e
                 }) || (null != (r = t.files) ? r.length : void 0) > 0 && ("copy" === (i = t.effectAllowed) || "all" === i) || /mac/i.test(navigator.platform) && /safari/i.test(navigator.userAgent))
             }, e.prototype.dragenter = function(e) {
                 return this._isDraggingFiles(e) ? (s.hide(), a.stop(e), this.didEnter = !0, c.defer(function(e) {
                     return function() {
                         return e.didEnter = !1
